expected-consensus.md

```diff
@@ -144,132 +145,158 @@ the prior block, or "losing" tickets generated in this epoch.
 
-Now, a miner must also check whether they have won in this round.
+To do so, the miner will use tickets from K blocks back as randomness to uniformly draw a value from 0 to 1. Comparing this value to their power, they determine whether they are eligible to mine. A user's `power` is defined as the ratio of the amount of storage they proved as of their last PoSt submission to the total storage in the network as of the current block.
 
-The process is as follows:
+Succinctly, the process of crafting a new `ElectionProof` in round `N` is as follows. We use:
 
-- At round N, the miner will draw the smallest ticket from the TipSet at round `N-K`, with `K` the randomness lookback parameter.
-  - In the case where there are multiple tickets to choose from at round `N-K` (i.e. if the TipSet eventually created has multiple blocks), miners should attempt to generate their `ElectionProof` from the ticket generated by the block with the smallest final ticket (i.e. not necessarily the smallest ticket generated at that round).
-- The miner will use this ticket as input to a VRF (Verifiable Random Function), thereby ensuring secrecy: no other participant can generate this output without the miner's private key. In that sense whether the miner has scratched a winning ticket remains a secret until they release their `ElectionProof` along with a new block.
-- The miner will compare the output to their power fraction from round `N-L`, with `L` the committee lookback parameter. If it is smaller, they have won and can mine a block inserting this winning `ElectionProof` in the block header for the block produced at round `N`. Else they wait to hear of another block generated in this round.
+- The ECVRF algorithm (must yield a pseudorandom, deterministic output) from [Goldberg et al. Section 5](https://tools.ietf.org/html/draft-irtf-cfrg-vrf-04#page-10), with:
+  - Sha256 for our hashing function
+  - Secp256k1 for our curve
+  - Note that the operation type in step 3.1 is not strictly necessary, but is used to distinguish this use of the VRF from that which generates tickets.
 
 At round N:
 
 ```text
-Input: Power Table, T_N, W, Map (deterministic function mapping a number to the range between 0 and 1)
+Input: parentTickets from N-K, miner's public key PK, miner's secret key SK, the Storage Market actor S
 Output: 1 or 0
 
-1. Determine total mining power = P
-2. Determine one's power fraction = p_f
-3. Determine if W is a winning lottery ticket
-  i. y <-- Map(pp,W) : {0,1}^n <-- pp x W
-  ii. If y < p_f then W is a winning lottery ticket
-    Return 1
-  iii. If y > p_f then W is not a winning lottery ticket
-    Return 0
+0. Prepare new election proof
+	i. newEP <-- New()
+1. Determine the miner's power fraction
+	i. 	# Determine total storage 1 round back
+  		S_n <-- storageMarket(N)
```

Comment:
spot check: will new state have been computed by this point? Will total power be accounted for in this round? or is this a fencepost?